### PR TITLE
Fix[15639563]: members dropdown menu issues

### PR DESCRIPTION
**Pull request by KornelKwiatkowski**
## Description

Task: https://tw.auroracreation.com/app/tasks/15639563

- Fixed issues with members dropdown menu

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

image

image

image

### CODE REVIEW

**Review by melyndav — CHANGES_REQUESTED** [review]
Hi @KornelKwiatkowski 

I made an error with the ticket in Team Work. When the user selects the search input, the text colour should stay as the text-gray-400 which is the placeholder text colour and not change to the text-gray-900 until the user starts typing.

My apologises for the mistake and the extra work. I will update the Team Work ticket.

**Review by rdig — CHANGES_REQUESTED** [review]
- **Please do a rebase pull, not a merge pull!**
Screenshot from 2023-11-08 14-59-31

- List shows incorrect avatars for users
Screenshot from 2023-11-08 14-26-02

- Bottom list padding is wrong
Screenshot from 2023-11-08 14-26-22

- Avatar is misaligned with the username
Screenshot from 2023-11-08 14-26-25

- Left list padding is wrong
Screenshot from 2023-11-08 14-26-30

- Placeholder text grammar is incorrect (enter vs. add)
Screenshot from 2023-11-08 14-26-58

- Input focus state changes the color of the placeholder text, which is incorrect
Screenshot from 2023-11-08 14-30-49

- Entering a non-existent username will show them in the list and allow you to click it
Screenshot from 2023-11-08 14-32-13

**Comment by KornelKwiatkowski** [comment]
"Please do a rebase pull, not a merge pull!"
@rdig My apologies, I read your comment in another PR after making this one.

**Comment by KornelKwiatkowski** [comment]
Thanks for the feedback @rdig. I appreciated this. I see all these issues, but they are out of scope of the task.
Please, check the link from Teamwork: https://tw.auroracreation.com/app/tasks/15639563

@melyndav I changed the placeholder color.

File: src/components/v5/common/Fields/InputBase/InputBase.tsx [hunk 1]
```diff
@@ -8,52 +8,58 @@ import { useAdjustInputWidth } from './hooks';
...
-  prefix,
-  suffix,
-  mode = 'primary',
-  disabled,
-  stateClassNames: stateClassNamesProp,
-  autoWidth = false,
-  ...rest
-}) => {
-  const stateClassNames = useStateClassNames(
+const InputBase = React.forwardRef<HTMLInputElement | null, InputBaseProps>(
```
Comment (joanna-pagepro) [inline src/components/v5/common/Fields/InputBase/InputBase.tsx:11]: I think that `| null` is redundant here

File: src/i18n/en.json [hunk 2]
```diff
@@ -1387,4 +1388,4 @@
     "errors.colonyObjective.title": "Colony objective title is required",
     "errors.colonyObjective.description": "Colony objective description is required",
     "errors.colonyObjective.progress": "Colony objective progress is required"
-}
+}
```
Comment (joanna-pagepro) [inline src/i18n/en.json:1391]: Please add the missing empty line

**Review by joanna-pagepro — CHANGES_REQUESTED** [review]

File: src/components/v5/common/Fields/InputBase/InputBase.tsx (hunk 1 above)
Comment (KornelKwiatkowski) [reply src/components/v5/common/Fields/InputBase/InputBase.tsx]: done

File: src/i18n/en.json (hunk 2 above)
Comment (KornelKwiatkowski) [reply src/i18n/en.json]: done

**Review by krawczykkatarzyna — DISMISSED** [review]

**Pull request closed without merging**